arm-resources/resources/2016-09-01/swagger/resources.json

```diff
@@ -1267,7 +1267,155 @@
         ],
         "responses": {
           "200": {
-            "description": "",
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/GenericResource"
+            }
+          }
+        }
+      }
+    },
+    "/{resourceId}": {
```

Comment:
can you also add the API for the following url:
`/{resourceId}/usages`

This API rightly fits under the generic ARM spec and not in the insights spec. They have no idea about the api-version that is supported by that RP.

/cc @gucalder - from the insights team.